apis/nucleus/src/components/listbox/components/EditableTitle.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useRef } from 'react';
+import { styled } from '@mui/material/styles';
+import { Typography, InputBase } from '@mui/material';
+import useProperties from '../../../hooks/useProperties';
+
+const Title = styled(Typography)(({ theme }) => ({
+  color: theme.listBox?.title?.main?.color,
+  fontSize: theme.listBox?.title?.main?.fontSize,
+  fontFamily: theme.listBox?.title?.main?.fontFamily,
+}));
+
+const StyledInput = styled(InputBase)(({ theme }) => ({
+  color: theme.listBox?.title?.main?.color,
+  fontSize: theme.listBox?.title?.main?.fontSize,
+  fontFamily: theme.listBox?.title?.main?.fontFamily,
```

Comment:
To keep the font while changing from `Title` to `StyledInput`
```suggestion
    fontSize: theme.listBox?.title?.main?.fontSize ?? theme.typography.h6.fontSize,
    fontFamily: theme.listBox?.title?.main?.fontFamily ?? theme.typography.h6.fontFamily,
    fontWeight: theme.listBox?.title?.main?.fontWeight ?? theme.typography.h6.fontWeight,
```

Reply:
Updated!